### DISCUSSION TITLE
Linting unused strings

### DISCUSSION THREAD

**Discussion by kieranm** [amannn/next-intl#398] [General]
Hey all,

I would be interested to know if there is some sort of way of checking for strings that are defined in config but unused in the codebase. Sometimes we refactor and remove parts of the app, but forget to go back and update the strings files.

Has anyone done something like this before, or do you have any ideas how it might be done?

Thanks
Kieran

**Comment by samuelstroschein** [comment]
You could use inlang's CLI https://inlang.com/documentation/apps/inlang-cli. The CLI exposes a lint command

**Comment by welfoz** [comment]
I'd be interested too! 
How did you solve this?

**Comment by amannn** [comment]
`i18n-check` is exploring detecting unused messages: [Validating messages](https://next-intl.dev/docs/workflows/messages)
